docs/adr/XXXX-use-pyenv-to-set-up-multiple-python-version-on-host.md

```diff
@@ -0,0 +1,29 @@
+# Use PyEnv to set up multiple Python version on host
+
+* Status: proposed
+* Date: 2020-04-07
+
+## Context and Problem Statement
+
+It is common that multiple versions of Python need to be install on development
+machines, which can be cumbersome.
+
+## Decision Drivers <!-- optional -->
+
+As a reminder, at time of writing, ODC only actively support linux, MacOS, and
+POSIX unix systems.
+
+## Considered Options
```

Comment:
Does `Anaconda` could also be on the list of considered options ?
Would be disqualified against pyenv as much more complex for this task only and some issues with transient dependencies (especially C++ ones) that make it less reliable.